Makefile

```diff
@@ -42,31 +42,31 @@ ifeq "$(strip $(VERSION))" ""
 	override VERSION = $(shell git describe --always --tags --dirty)
 endif
 
-LDFLAGS_linux = -static
-LDFLAGS_darwin =
-LDFLAGS_windows =
-
-GO_BUILD_TAGS_linux = "osusergo netgo static_build release"
-GO_BUILD_TAGS_darwin = "release"
-GO_BUILD_TAGS_windows = "release"
-
 GO_LDFLAGS = -X $(VERSION_PACKAGE).version=$(VERSION)
 GO_LDFLAGS += -X $(VERSION_PACKAGE).buildDate=$(shell date +'%Y-%m-%dT%H:%M:%SZ')
 GO_LDFLAGS += -X $(VERSION_PACKAGE).gitCommit=$(COMMIT)
 GO_LDFLAGS += -X $(VERSION_PACKAGE).gitTreeState=$(if $(shell git status --porcelain),dirty,clean)
 GO_LDFLAGS += -s -w
 
-GO_LDFLAGS_windows =" $(GO_LDFLAGS)  -extldflags \"$(LDFLAGS_windows)\""
-GO_LDFLAGS_darwin =" $(GO_LDFLAGS)  -extldflags \"$(LDFLAGS_darwin)\""
-GO_LDFLAGS_linux =" $(GO_LDFLAGS)  -extldflags \"$(LDFLAGS_linux)\""
+GO_BUILD_TAGS_linux = osusergo netgo static_build release
+LDFLAGS_linux = -static
+
+GO_BUILD_TAGS_windows = release
+
+# darwin/arm64 requires Go 1.16beta1 or later; dockercore/golang-cross
+# doesn't have a recent macOS toolchain so disable CGO and use
+# github.com/rjeczalik/notify's kqueue support. 
+GO_VERSION_darwin_arm64 = 1.16beta1
+CGO_ENABLED_darwin_arm64 = 0
+GO_BUILD_TAGS_darwin = release
 
 ifneq "$(strip $(LOCAL))" "true"
 	override STATIK_FILES =  cmd/skaffold/app/cmd/statik/statik.go
 endif
 
 # when build for local development (`LOCAL=true make install` can skip license check)
 $(BUILD_DIR)/$(PROJECT): $(STATIK_FILES) $(GO_FILES) $(BUILD_DIR)
-	GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=1 go build -gcflags="all=-N -l" -tags $(GO_BUILD_TAGS_$(GOOS)) -ldflags $(GO_LDFLAGS_$(GOOS)) -o $@ $(BUILD_PACKAGE)
+	GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=1 go build -gcflags="all=-N -l" -tags "$(GO_BUILD_TAGS_$(GOOS))" -ldflags "$(GO_LDFLAGS_$(GOOS))" -o $@ $(BUILD_PACKAGE)
```

Comment:
```GO_LDFLAGS_$(GOOS)``` doesn't seem to be defined for windows, darwin, or linux after these changes, are they supposed to be defined somewhere here?

Reply:
Hmm you're right — I've missed something here.